mantidproject/mantid
# Add Mass as a parameter rather than attribute

This moves the `Mass`attribute of the `ComptonProfile` fit function (which bases `GaussianComptonProfile` and `GramCharlierComptonProfile`) from being an attribute to a fitted parameter.

Existing usage is replaced with the parameter being tied.

**To test:**

Run a Vesuvio fitting script and see that the correct masses are used (log output from `VesuvioResolution` function is a good indicator).

Fixes #15669.

Does not need to be in the release notes.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
